### PR TITLE
chore: switch to rimraf & cross-env for windows support

### PR DESCRIPTION
**Pull request by Skye-31**
Related: #1674 

Not sure if a changeset is needed for this? Happy to remove if not desired

### CODE REVIEW

**Comment by Cherry** [comment]
Thanks @Skye-31!

It looks like https://github.com/cloudflare/wrangler2/blob/main/package.json already lists `cross-env`, but `rimraf` should be added as least as `devDependencies` here too I think. Tests must be passing due to them being transitive dependencies of something else right now. 😅

File: .changeset/quiet-readers-smell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+chore: use rimraf & cross-env to support windows development
```
Comment (GregBrimble) [inline .changeset/quiet-readers-smell.md:5]: Yeah, not required, but not harmful to have either. Take it or leave it.

**Comment by Skye-31** [comment]
Good catch!

**Review by GregBrimble — APPROVED** [review]
I think you've got an extra space after `rimraf` now, but LGTM

**Comment by Skye-31** [comment]
Ah yes, I blame that on the GitHub "commit suggestion" thing!

**Pull request merged**